exonum-java-binding-parent/pom.xml

```diff
@@ -0,0 +1,647 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Git kind of breaks in such cases of moving and replacing original file, showing this one as brand new :)